packaging/docker/centos-linux/Dockerfile

```diff
@@ -1,24 +1,28 @@
-FROM centos:7.7.1908
+FROM centos:8.3.2011
```

Comment:
I'm not very familiar with centos, but is jumping from 7.x to 8.x significant?

Assuming 8.x is stable then I don't see a problem with it (we recently bumped alpine and  ubuntu by major versions as well), but it might be worth calling out in the PR description to raise the visibility of the change.

Reply:
CentOS 8.x is their current stable release: I see no reason not to bump it. I found no issues with it either.